docs/content/release_notes.rst

```diff
@@ -1,5 +1,25 @@
 Release Notes
 =============
+Release 0.5.0 - 2021/07/15
+--------------------------
+
+(Major Release)
+
+**New Application** - Octree Mesh Creation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+New application for the creation of octree meshes with arbitrary refinements around entities.
+
+Application Updates
+^^^^^^^^^^^^^^^^^^^
+
+- Major update to the Peak Finder algorithm and application. New selection panel for the query of channel groups. Faster plotting.
+- Implementation of the ui.json for Peak Finder and Octree Mesh Creation. Parameters can be re-loaded in the
```

Comment:
Should we have a note about running from GA?

Reply:
I thought of keeping it on the down-low until 3.3.1 is released?

Reply:
Yep.  Fair enough - let's iron out kinks first!

Reply:
Octree mesh app discusses drag and drop for 3.3.1 - was this intended?  Are we just keeping the peakfinder ui.json drag and drop on the dl?

Reply:
Ok fair enough. I will add a note.